SoftLayer/CLI/account/event_detail.py

```diff
@@ -0,0 +1,72 @@
+"""Details of a specific event, and ability to acknowledge event."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.account import AccountManager as AccountManager
+from SoftLayer import utils
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--ack', is_flag=True, default=False,
+              help="Acknowledge Event. Doing so will turn off the popup in the control portal")
+@environment.pass_env
+def cli(env, identifier, ack):
+    """Details of a specific event, and ability to acknowledge event."""
+
+    # Print a list of all on going maintenance
+    manager = AccountManager(env.client)
+    event = manager.get_event(identifier)
+
+    if ack:
+        manager.ack_event(identifier)
+
+    env.fout(basic_event_table(event))
+    env.fout(impacted_table(event))
+    env.fout(update_table(event))
+
+
+def basic_event_table(event):
+    """Formats a basic event table"""
+    table = formatting.Table(["Id", "Status", "Type", "Start", "End"], title=event.get('subject'))
```

Comment:
The title lost its shape because the string has escape sequences \r, \n, \t, etc.
![image](https://user-images.githubusercontent.com/6937170/54844999-9d625b00-4cae-11e9-8d8c-6b9878060672.png)